docs/howto/openstack/neutron/vpnaas.md

```diff
@@ -0,0 +1,786 @@
+`server-kna1` (in `kna1`). With the VPN connection between the two regions
+now disabled, pinging should not be possible:
+
+```console
+ubuntu@server-fra1:~$ ping -c 3 10.15.20.148
+PING 10.15.20.148 (10.15.20.148) 56(84) bytes of data.
+
+--- 10.15.20.148 ping statistics ---
+3 packets transmitted, 0 received, 100% packet loss, time 2055ms
+```
+
+```console
+ubuntu@server-kna1:~$ ping -c 3 10.15.25.58
+PING 10.15.25.58 (10.15.25.58) 56(84) bytes of data.
+
+--- 10.15.25.58 ping statistics ---
+3 packets transmitted, 0 received, 100% packet loss, time 2050ms
+```
+
+## Enabling an inactive connection
+
+You can easily enable an inactive site-to-site VPN connection.
+
+=== "{{gui}}"
+    Currently, there is no way to enable an inactive connection from 
+    the {{gui}}. If you want to disable an inactive connection, please use 
+    the OpenStack CLI.
+=== "OpenStack CLI"
+    Make sure you hop on the side where you initially disabled the 
+    connection. According to the example scenario we described in the 
+    previous section, that would be the left side (region `fra1`), and the 
+    name of the disabled connection would be `vpn-conn-to-kna1`. Make sure 
+    the connection status is `DOWN`:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status
+    ```
+        
+    ```plain
+    +--------+-------+
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+        
+    Note that if you issued a similar command from the right side of 
+    the connection (region `kna1`), you would also get a `DOWN` status. 
+    Being on the left side, all you have to do to enable the inactive 
+    connection is type the following:
+        
+    ```bash
+    openstack vpn ipsec site connection set --enable vpn-conn-to-kna1
+    ```
+        
+    Check the connection status --- it should be `ACTIVE`:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status
+    ```
+        
+    ```plain
+    +--------+--------+
+    | Field  | Value  |
+    +--------+--------+
+    | Status | ACTIVE |
+    +--------+--------+
+    ```
+        
+    You get the same status by issuing a similar command from the right 
+    side.
+        
+    > Again, since you may have to wait several seconds before seeing 
+    the status change you expect, try something like this: 
+    ```bash
+    watch "openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status"
+    ``` 
+    Hit CTRL+C to stop watching.
+
+To test that a previously disabled site-to-site connection is now enabled,
+select a server from one region and try to ping a server in another region.
+Suppose, for example, that a re-enabled connection involves regions
+`fra1` and `kna1`, and that we have servers `server-fra1` (in `fra1`) and
+`server-kna1` (in `kna1`). With the VPN connection between the two regions
+now re-established, pinging should be all possible:
```

Comment:
I think the word "all" can be removed here.

Reply:
It sure can :)